pkg/resource/deploy/step.go

```diff
@@ -993,7 +1051,7 @@ func (s *ImportStep) Apply(preview bool) (resource.Status, StepCompleteFunc, err
 	s.old = resource.NewState(s.new.Type, s.new.URN, s.new.Custom, false, s.new.ID, read.Inputs, read.Outputs,
 		s.new.Parent, s.new.Protect, false, s.new.Dependencies, s.new.InitErrors, s.new.Provider,
 		s.new.PropertyDependencies, false, nil, nil, &s.new.CustomTimeouts, s.new.ImportID, s.new.RetainOnDelete,
-		s.new.DeletedWith)
+		s.new.DeletedWith, nil, nil)
```

Comment:
I think this could take s.new.Created/Modified but not sure it would actually change anything. Might be easier to reason about later on that this is just a copy of new and the nil timestamps aren't important.